src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants.java

```diff
@@ -0,0 +1,3865 @@
+    String newMarkdownDocMenuLabel();
+    @DefaultStringValue("Create a new Markdown document") // $NON-NLS-1$
+    String newMarkdownDocDesc();
+    
+    // newPythonDoc
+    @DefaultStringValue("_Python Script") // $NON-NLS-1$
+    String newPythonDocMenuLabel();
+    @DefaultStringValue("Create a new Python script") // $NON-NLS-1$
+    String newPythonDocDesc();
+    
+    // newShellDoc
+    @DefaultStringValue("_Shell Script") // $NON-NLS-1$
+    String newShellDocMenuLabel();
+    @DefaultStringValue("Create a new shell script") // $NON-NLS-1$
+    String newShellDocDesc();
+    
+    // newStanDoc
+    @DefaultStringValue("_Stan File") // $NON-NLS-1$
+    String newStanDocMenuLabel();
+    @DefaultStringValue("Create a new Stan program") // $NON-NLS-1$
+    String newStanDocDesc();
+    
+    // newHtmlDoc
+    @DefaultStringValue("_HTML File") // $NON-NLS-1$
+    String newHtmlDocMenuLabel();
+    @DefaultStringValue("Create a new HTML file") // $NON-NLS-1$
+    String newHtmlDocDesc();
+    
+    // newJavaScriptDoc
+    @DefaultStringValue("_JavaScript File") // $NON-NLS-1$
+    String newJavaScriptDocMenuLabel();
+    @DefaultStringValue("Create a new JavaScript file") // $NON-NLS-1$
+    String newJavaScriptDocDesc();
+    
+    // newCssDoc
+    @DefaultStringValue("_CSS File") // $NON-NLS-1$
+    String newCssDocMenuLabel();
+    @DefaultStringValue("Create a new CSS file") // $NON-NLS-1$
+    String newCssDocDesc();
+    
+    // newD3Doc
+    @DefaultStringValue("_D3 Script") // $NON-NLS-1$
+    String newD3DocMenuLabel();
+    @DefaultStringValue("Create a new D3 Script") // $NON-NLS-1$
+    String newD3DocDesc();
+    
+    // newRPlumberDoc
+    @DefaultStringValue("Plumber _API...") // $NON-NLS-1$
+    String newRPlumberDocMenuLabel();
+    @DefaultStringValue("Create a new Plumber API") // $NON-NLS-1$
+    String newRPlumberDocDesc();
+    
+    // rcppHelp
+    @DefaultStringValue("Help on using Rcpp") // $NON-NLS-1$
+    String rcppHelpDesc();
+    
+    // printCppCompletions
+    @DefaultStringValue("Print C++ Completions") // $NON-NLS-1$
+    String printCppCompletionsDesc();
+    
+    // newSweaveDoc
+    @DefaultStringValue("R _Sweave") // $NON-NLS-1$
+    String newSweaveDocMenuLabel();
+    @DefaultStringValue("Create a new R Sweave document") // $NON-NLS-1$
+    String newSweaveDocDesc();
+    
+    // newRMarkdownDoc
+    @DefaultStringValue("R _Markdown...") // $NON-NLS-1$
+    String newRMarkdownDocMenuLabel();
+    @DefaultStringValue("Create a new R Markdown document") // $NON-NLS-1$
+    String newRMarkdownDocDesc();
+    
+    // newQuartoDoc
+    @DefaultStringValue("_Quarto Doc...") // $NON-NLS-1$
+    String newQuartoDocMenuLabel();
+    @DefaultStringValue("Create a new Quarto document") // $NON-NLS-1$
+    String newQuartoDocDesc();
+    
+    // serveQuartoSite
+    @DefaultStringValue("_Serve Site") // $NON-NLS-1$
+    String serveQuartoSiteMenuLabel();
```

Comment:
`serveQuartoSiteMenuLabel` and `serveQuartoSiteDesc` are both defined twice in this file, so it will not compile. Not sure if this was a merge conflict or something invalid output by the tool.

Reply:
Ok, I'll take over getting this change tweaked and merged. Thanks!